docs/sidebar.md

```diff
@@ -57,7 +57,7 @@
          - [Install Merlin in airgap environment](./guides/platform/Install_MERLIN_In_AirGap.md)
       - [Update Merlin CA and certificates](./guides/platform/Update_CA_and_Certificate.md)
       - [Manage browser certificates](./guides/platform/Manage_browser_certs.md)
-      - [Upgrade Merlin platform and tools](./guides/platform/Upgrade_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md)
+      - [Upgrade to v2](./guides/platform/Upgrade_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md)
```

Comment:
use 'Upgrade from Version 1 to Version 2' instead of v2
what about v2 updates documentation?

Reply:
Done

Reply:
Yes we have [https://ibm.github.io/merlin-docs/#/./guides/platform/Update_catalogsource](https://ibm.github.io/merlin-docs/#/./guides/platform/Update_catalogsource) and [https://ibm.github.io/merlin-docs/#/./guides/platform/upgrade_merlin_operator](https://ibm.github.io/merlin-docs/#/./guides/platform/upgrade_merlin_operator)